### PR TITLE
3 995 Egyptian hieroglyphs

### PR DESCRIPTION
**Pull request by eggrobin**
UTC-176-C29 Consensus: Provisionally assign 3,994 code points U+13460..U+143F9 in a new Egyptian Hieroglyphs Extended-A block at U+13460..U+143FF for 3,994 Egyptian Hieroglyph characters as described in [L2/23-181](https://www.unicode.org/cgi-bin/GetMatchingDocs.pl?L2/23-181). (Reference: Section 1 of [L2/23-164](https://www.unicode.org/cgi-bin/GetMatchingDocs.pl?L2/23-164))

+U+143FA from L2/23-181R.

Note that per the proposal, we are doing UnicodeData.txt the Khitan small script way, not the Tangut or CJKV way.

UTC-178-C?? Consensus: Approve 3,995 Egyptian Hieroglyph characters U+13460..U+143FA in a new Egyptian Hieroglyphs Extended-A block at U+13460..U+143FF, as described in L2/23-181R for Unicode Version 16.0 [Reference: Section 2a of L2/24-013]

### CODE REVIEW

**Review by markusicu — COMMENTED** [review]
lgtm but my understanding is that Michel will prepare a new version of the proposal doc which inserts one additional character, so most of the files will need to change again

**Review by markusicu — COMMENTED** [review]
lgtm pending approval at UTC-178

File: unicodetools/data/ucd/dev/VerticalOrientation.txt
Comment (eggrobin) [inline unicodetools/data/ucd/dev/VerticalOrientation.txt:2143]: doh

**Review by markusicu — COMMENTED** [review]
lgtm2

**Review by markusicu — APPROVED** [review]

**Pull request merged**
